doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1611,12 +1612,10 @@ pg_restore -h <remote-db-url> -p <remote-db-port> -U centerui_admin -O -n center
 8. Update `/etc/xroad/db.properties` contents with correct database host URLs and passwords.
 
 ```properties
-    username=centerui
-    password=<centerui password>
-    database=centerui_production
-    host=<database host>
-    port=<database port>
-    schema=centerui
+    spring.datasource.username=centerui
+    spring.datasource.password=<centerui password>
+    spring.datasource.hikari.data-source-properties.currentSchema=centerui
+    spring.datasource.url=jdbc:postgresql://<database host>:<database port>/centerui_production
```

Comment:
The previous comments apply to here too. It would be good to keep different parts of the documentation consistent.